server/controllers/finance/reports/creditors/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @overview finance/reports/creditors/index.js
+ *
+ * @description
+ * This report displays the amounts owed by creditor groups broken down by age of
+ * their debt.  The report highlights clients who have long overdue debts, so
```

Comment:
Some of this comment doesn't make any sense since it was lifted from the Aged Debtors report.  You can just remove everything after the first sentence.